src/vuex/modules/common/kuzzle/store.ts

```diff
@@ -31,6 +31,13 @@ export const envColors = [
   'magenta'
 ]
 
+const wait = async ms =>
+  new Promise(resolve => {
+    setTimeout(() => {
+      resolve()
+    }, ms)
+  })
+
 const checkEnvironment = e => {
```

Comment:
missing async since their are await statement in this fonction

Reply:
Where do you see await statements in this function?

Reply:
@rolljee in splited code review mode you need to unfold parts sometimes. The await statements you seen is part of 
```js
async switchEnvironment(context, id) {
```